syntax = "proto3";

package prodisdb.protobuf;

option go_package = "gitlab.com/prodisdb/protobuf/db";
option csharp_namespace = "ProdisDB.Protobuf";
option java_package = "com.prodisdb.protobuf";
option objc_class_prefix = "PDB";

import "google/protobuf/any.proto";

service Db {
  rpc Get (GetReq) returns (GetResp);

  rpc Put (PutReq) returns (PutResp);

  rpc Delete (DelReq) returns (DelResp);
}

message GetReq {
}

message GetResp {
}

message PutReq {
  google.protobuf.Any serializable = 1;
}

message PutResp {
}

message DelReq {
}

message DelResp {
}
